docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+    myapp:
+        build: .
+        container_name: "myapp"
+        image: debian/latest
```

Comment:
The image is specified in the Dockerfile, is there a reason to specify a different one here?